vmdb/app/models/vm_scan.rb

```diff
@@ -1,4 +1,17 @@
 class VmScan < Job
+
+  #
+  # TODO: until we get location/offset read capability for OpenStack
+  # image data, OpenStack fleecing is prone to timeout (based on image size).
+  # We adjust the queue timeout in server_smart_proxy.rb, but that's not enough,
+  # we also need to adjust the job timeout here.
+  #
+  DEFAULT_TIMEOUT = defined?(RSpec) ? 300 : 3000
```

Comment:
Why is the RSpec check needed?  I still don't understand why it's needed if the tests use Timecop.